cli/cmd/cluster.go

```diff
@@ -249,31 +253,20 @@ func promptForEmail() {
 }
 
 func refreshCachedClusterConfig(awsCreds *AWSCredentials) *clusterconfig.ClusterConfig {
-	userClusterConfig := &clusterconfig.ClusterConfig{}
-	err := clusterconfig.SetDefaults(userClusterConfig)
+	accessClusterConfig, err := getAccessClusterConfig()
 	if err != nil {
 		exit.Error(err)
 	}
 
-	if flagClusterConfig != "" {
-		err := readUserClusterConfigFile(userClusterConfig)
-		if err != nil {
-			exit.Error(err)
-		}
+	// add empty file if cached cluster doesn't exist so that the file output by manager container maintains current user permissions
+	cachedConfigPath := cachedClusterConfigPath(*accessClusterConfig.ClusterName, *accessClusterConfig.Region)
```

Comment:
This appears to be the same as `mountedConfigPath` variable in line 267, delete one of them? Or perhaps there is a type on line 267 and it should be `mountedConfigPath := mountedClusterConfigPath(...`

Reply:
Yes that is a typo, good catch!